.azure-pipelines.yml

```diff
@@ -27,7 +27,7 @@ variables:
 
 stages:
 - stage: build_test
-  displayName: Build and Test
+  displayName: Run
```

Comment:
As you say it's probably not very important, about "Run" in the Docker context could be misinterpreted as "running the Docker image". So how about "Check" as a name? And should we also rename the non-display stage name?

Reply:
I think in all cases but the Docker job "Run" sounds better, e.g. "Run LinuxTest" vs. "Check LinuxTest". I'd rather rename the Docker job to "DockerBuild". I'll update the stage name when we have a decision here.

Reply:
I like "DockerBuild".

Reply:
Done.